fml/platform/android/jni_util.cc

```diff
@@ -40,6 +54,9 @@ JNIEnv* AttachCurrentThread() {
   }
   jint ret = g_jvm->AttachCurrentThread(&env, &args);
   FML_DCHECK(JNI_OK == ret);
+
+  tls_jni_detach.reset(new JNIDetach());
```

Comment:
Should we dcheck that this isn't already somehow set, immediately detaching us?

Reply:
That should not be possible - if the thread is already attached, then the call to JavaVM::GetEnv above will succeed.  Added an assert just in case.